2-positioning/styles/styles.css

```
@@ -0,0 +1,33 @@
+header {
+    color: white;
+    background-color: black;
+    border: 3px solid green;
+    position: sticky;
+}
+
+#menu {
+    border: 3px solid blue;
+    position: absolute;
+    width: 12.5%;
```

Follow the requirements please.
```suggestion
    width: 200px;
```